### ISSUE TITLE
inspector breaks for Glimmer Components containing {{link-to}}

### ISSUE THREAD

**Issue by mike-north** [emberjs/ember-inspector#961]
## Instructions to Reproduce

```sh
ember new -b @ember/octane-app-blueprint my-app
cd my-app
ember s
ember g component x-foo
```

Define the component as follows

#### app/components/x-foo.js (unaltered)
```js
import Component from '@glimmer/component';

export default class XFooComponent extends Component {
}
```
#### app/templates/components/x-foo.hbs
```hbs
{{link-to 'Go Home' 'index'}}
```

Finally.... open the ember inspector, and select the "Components" tab. The error message below seems to be logged each time the tab is clicked (i.e., alternating back and forth between components and another tab will cause the error to be logged repeatedly)

#### Is it really the inspector?

The error goes away if the extension is removed, or if the "Components" inspector tab is never used

#### Versions

Ember Inspector (chrome) v3.6.0 (Updated January 29, 2019) running on Brave v0.62.51 (Chromium: 73.0.3683.103)

<details>
   <summary> package.json </summary>

```json
{
  "name": "my-app",
  "version": "0.0.0",
  "private": true,
  "description": "Small description for my-app goes here",
  "repository": "",
  "license": "MIT",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build",
    "lint:hbs": "ember-template-lint .",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "babel-eslint": "^8.0.2",
    "@ember/optional-features": "^0.7.0",
    "@glimmer/component": "^0.14.0-alpha.3",
    "broccoli-asset-rev": "^3.0.0",
    "ember-auto-import": "^1.2.20",
    "ember-cli": "github:ember-cli/ember-cli#1dc682ea479b084d7f03215c6c554138ed4bf76b",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "^7.7.0",
    "ember-cli-dependency-checker": "^3.0.0",
    "ember-cli-eslint": "^5.0.0",
    "ember-cli-htmlbars": "^3.0.0",
    "ember-cli-htmlbars-inline-precompile": "^2.0.0",
    "ember-cli-inject-live-reload": "^2.0.1",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-template-lint": "^1.0.0-beta.1",
    "ember-cli-uglify": "^2.1.0",
    "ember-data": "github:emberjs/data#1df833396855d956b817540923dd89338463fec2",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^2.0.0",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-qunit": "^4.1.2",
    "ember-resolver": "^5.1.3",
    "ember-source": "https://s3.amazonaws.com/builds.emberjs.com/canary/shas/06c0cd0176e55189f375d25daa7500b05c16e4b1.tgz",
    "ember-welcome-page": "^3.2.0",
    "eslint-plugin-ember": "^6.0.1",
    "eslint-plugin-node": "^8.0.1",
    "loader.js": "^4.7.0",
    "qunit-dom": "^0.8.0"
  },
  "engines": {
    "node": "6.* || 8.* || >= 10.*"
  }
}
```

</details>

`yarn.lock` available upon request

#### Error Message
```
VM8521:1646 Uncaught TypeError: component.get is not a function
    at exports.default.controllerForComponent (<anonymous>:1646:34)
    at exports.default.controllerForComponent (<anonymous>:1654:21)
    at <anonymous>:1332:38
    at Array.forEach (<anonymous>)
    at exports.default.componentsForController (<anonymous>:1331:18)
    at <anonymous>:1299:31
    at Array.map (<anonymous>)
    at exports.default.buildComponentTrees (<anonymous>:1298:26)
    at exports.default.build (<anonymous>:1219:61)
    at Class.viewTree (<anonymous>:4705:33)
```

**Comment by rwwagner90** [comment]
@mike-north thanks for the incredibly detailed issue here! Template only components and glimmer components do not currently work with inspector, I don't think. We definitely will have to figure this out before Octane drops.

**Comment by mike-north** [comment]
> Template only components and glimmer components do not currently work with inspector

**Comment by rwwagner90** [comment]
@rwjblue we'll need the component manager changes to get this working, right?

**Comment by rwjblue** [comment]
Yes, I think so but I definitely think we can fix the specific error reported here independently of that.

**Comment by mike-north** [comment]
Repro case 

**Code:** https://codesandbox.io/s/github/mike-north/octane-app-repro-cases/tree/hbs-debugger
**Fullscreen App:** https://jll45pq3pw.sse.codesandbox.io/

**Comment by bitwolfe** [comment]
What's the status of this? Seems to still be an issue. We're getting quite close to Octane release...

**Comment by lolmaus** [comment]
Is there at least some workaround?

I don't mind parts of Inspector not working, but Inspector crashing the app and breaking tests — this is 😭.

**Comment by rwwagner90** [comment]
@lolmaus what Ember version are you on?

**Comment by lolmaus** [comment]
@rwwagner90 `ember-source@3.13.0-beta.5`

**Comment by rwwagner90** [comment]
The new stuff will be enabled for 3.14+, when Octane drops. I'm not sure if we should support glimmer components in versions before Octane.

**Comment by chancancode** [comment]
Should be fixed by #1088 in the next release.
